can not update to or install atom-typescript v10.1.10

```
% apm --version
apm  1.13.0
npm  2.15.9
node 4.6.0
python 2.7.12
git 2.7.4
```

```
% apm install atom-typescript
Installing atom-typescript to /home/$MYLOGINNAME/.atom/packages ✗

npm WARN peerDependencies The peer dependency typescript@* included from byots will no
npm WARN peerDependencies longer be automatically installed to fulfill the peerDependency 
npm WARN peerDependencies in npm 3+. Your application will need to depend on it explicitly.
npm WARN optional dep failed, continuing fsevents@1.0.14
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm ERR! Linux 4.7.7-200.fc24.x86_64
npm ERR! argv "/usr/lib/node_modules/atom-package-manager/bin/node" "/usr/lib/node_modules/npm/bin/npm-cli.js" "--globalconfig" "/home/$MYLOGINNAME/.atom/.apm/.apmrc" "--userconfig" "/home/$MYLOGINNAME/.atom/.apmrc" "install" "/tmp/d-116929-3503-1jawem1/package.tgz" "--runtime=electron" "--target=0.37.8" "--arch=x64" "--global-style" "--legacy-bundling"
npm ERR! node v4.6.0
npm ERR! npm  v2.15.9
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package typescript@2.1.0-dev.20161023 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer byots@2.1.0-dev.20161022.0.26 wants typescript@*

npm ERR! Please include the following file with any support request:
npm ERR!     /tmp/apm-install-dir-116929-3503-1c1yv7o/npm-debug.log
```

there is no directory `/tmp/apm-install-dir-116929-3503-1c1yv7o`

maybe the problem is `peerDependencies` from [byots](https://github.com/basarat/byots/blob/master/package.json#L29)

```
"typescript": "*"
```

`*` doesn't match pre-release versions

```
% semver -r '*' 2.0.0 2.0.1 2.1.0 2.1.1 2.1.0-dev.20161023 2.2.0 2.2.1-dev.20161023 2.3.0
2.0.0
2.0.1
2.1.0
2.1.1
2.2.0
2.3.0
```

documentation for [node-semver](https://github.com/npm/node-semver#prerelease-tags) says

> If a version has a prerelease tag (for example, 1.2.3-alpha.3) then it will only be allowed to satisfy comparator sets if at least one comparator with the same [major, minor, patch] tuple also has a prerelease tag.

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "atom-space-pen-views": "^2.0.4",
     "babel": "^5.6.23",
     "basarat-text-buffer": "6.0.0",
-    "byots": "2.1.0-dev.20161022.0.26",
+    "byots": "2.1.0-dev.20161029.21.55",
     "d3": "^3.5.5",
     "detect-indent": "^4.0.0",
     "detect-newline": "^2.1.0",

